thentos-adhocracy/src/Thentos/Adhocracy3/Backend/Api/Simple.hs

```diff
@@ -452,6 +486,9 @@ extractUserId resp = do
         (Aeson.eitherDecode . Client.responseBody $ resp :: Either String TypedPath)
     userIdFromPath $ tpPath resource
 
+sendRequest :: Client.Request -> IO (Client.Response LBS)
+sendRequest req = Client.withManager Client.defaultManagerSettings $ Client.httpLbs req
```

Comment:
The documentation for the `http-client` library says using a new manager for each request to the same host isn't great (also `withManager` is being deprecated). We already have a manager being threaded around - can't we use that one?